### PR TITLE
💎(site) Add New Design to Technical Landing - Body

### LINKED ISSUE #291: Landing Page: Technical
Design: https://www.figma.com/file/xtBKZcRBo7H2Jrjs7DjNqE/Final-Designs?node-id=2%3A34151

### LINKED ISSUE #339: Module: Content
Technical Landing Page :: Content area

- Section with cards 
- Section with message

Design: https://www.figma.com/file/xtBKZcRBo7H2Jrjs7DjNqE/Final-Designs?node-id=2%3A34151

### PR DESCRIPTION
**Pull request by Vanss472**
- added interest form to tech detail template
- added get started section to tech detail template

**Please tag each issue this Pull Request resolves**

- Resolves #339 
- Resolves #291

@Snugug my bad here I pushed the Technical content changes to the **content/getting-started** branch directly. Let me know if you need me to reverse it. 

Also about the interest form section. In my opinion, it looks better if the CTA is full width rather than what you recommended.

Screenshot if side-by-side:
Screen Shot 2022-04-06 at 11 52 10 AM

vs full width:
image

### CODE REVIEW

**Review by Snugug — CHANGES_REQUESTED** [review]
Thanks for this! I've got a few small comments, but otherwise it looks great!

File: lib/linting/schemas/technical.js [hunk 1]
```diff
@@ -31,14 +31,85 @@ module.exports = {
...
           description: 'Weight of article. Lower raises it in lists',
         },
+        started: {
+          type: 'object',
+          description: 'Get started section',
+          properties: {
+            title: {
+              type: 'string',
+              description: 'Title to display in get started section',
+              minLength: 10,
```
[the remark this answers is not in the document]
Comment (Vanss472) [reply lib/linting/schemas/technical.js:41]: set to `maxLength: 500,`

File: site/sass/layouts/_tech-detail.scss [hunk 2]
```diff
@@ -92,6 +92,60 @@
...
+  &__interest-form {
+    background-color: var(--grey-50);
+    border-radius: 0.625rem;
+    margin: $spacing auto;
+    max-width: 60ch;
+    padding: 2rem 2rem 2.5rem;
+
+    p {
+      color: var(--grey-700);
+    }
```
[the remark this answers is not in the document]
Comment (Vanss472) [reply site/sass/layouts/_tech-detail.scss:146]: fixed

File: site/sass/layouts/_tech-detail.scss [hunk 3]
```diff
@@ -92,6 +92,60 @@
...
+      height: 100%;
+      padding: 2.5rem;
+      position: relative;
+      text-align: left;
+      width: 100%;
+      z-index: 1;
+
+      p {
+        color: var(--grey-700);
+      }
```
[the remark this answers is not in the document]
Comment (Vanss472) [reply site/sass/layouts/_tech-detail.scss:129]: fixed

File: site/_layouts/tech-detail.njk [hunk 4]
```diff
@@ -32,7 +32,32 @@ layout: _wrapper
...
+      {% if started.desc %}
+      <p class="type--large tech-detail__get-started--desc">{{started.desc}}</p>
+      {% endif %}
+      <div class="tech-detail__get-started--grid">
+        {% for card in started.cards %}
+        <div class="tech-detail__get-started--grid-item">
+          <header class="tech-detail__get-started--heading">
+            <h3 class="type--h4">{{ card.title }}</h3>
+          </header>
+          {{ card.body | markdown | safe }}
```
[the remark this answers is not in the document]
Comment (Vanss472) [reply site/_layouts/tech-detail.njk:47]: done.

**Review by Snugug — DISMISSED** [review]

**Comment by Snugug** [comment]
Thanks for your work here! I'm calling this _done_! I'm not going to merge yet, as I want to update our other sections with getting started content.

**Review by Snugug — APPROVED** [review]

**Pull request merged**